apps/frontend/src/lib/components/LanguageSwitcher.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+	import { locale, availableLocales } from '$lib/stores/locale';
+	import { setLocale, type Locale } from '$lib/paraglide/runtime.js';
+	import { Button, Dropdown, DropdownItem } from 'flowbite-svelte';
+	import { ChevronDownOutline } from 'flowbite-svelte-icons';
+
+	const localeNames: Record<string, string> = {
+		en: 'English',
+		nl: 'Dutch',
```

Comment:
```suggestion
		nl: 'Nederlands',
```

Needs to be in the native language.